python-main.js

```diff
@@ -1258,22 +1258,11 @@ function web_editor(config) {
         webusbErrorHandler(error);
     }
 
-    function clearDapWrapper(event) {
-        if(window.dapwrapper || window.previousDapWrapper) {
-            window.dapwrapper = null;
-            window.previousDapWrapper = null;
-        }
-    }
-
     function doConnect(serial) {
         // Change button to connecting
         $("#command-connect").hide();
         $("#command-connecting").show();
         $("#command-disconnect").hide();
-        
-        // Device disconnect listener
-        // Clears dapwrapper
-        navigator.usb.addEventListener('disconnect', clearDapWrapper);
```

Comment:
Removed from here since the disconnect event already executes showDisconnectError().

showDisconnectError() -> webusbErrorHandler() -> clearDapWrapper()